scripts/create_policy

```diff
@@ -129,15 +131,46 @@ def process_ima_buf_in_measurement_list(
     return keyrings_map, ima_buf_map, ret
 
 
+def process_signature_verification_keys(verification_keys, policy):
+    """Add the given keys (x509 certificates) to keyring"""
+
+    verification_key_list = None
+    if verification_keys:
+        if policy.get("verification-keys", None):
```

Comment:
If `policy` is a `dict`, `None` can be elided

Reply:
Fixed.